### DISCUSSION TITLE
Drug Type Measurements and Unit of Measure

### DISCUSSION THREAD

**Discussion by zprice22** [CMSgov/hospital-price-transparency#19] [Q&A] [answered]
The drug type values are noted to be based on industry standards (e.g., NDC). The values appear to reflect volume measurements (e.g., ML) and are not consistent with HCPCS billable units (e.g., MCG). 

For example, Octreotide is billed using HCPCS J2354, "Injection, octreotide, non-depot form for subcutaneous or intravenous injection, 25 mcg". The hospital dispenses NDC 0078018001 as packaged as 50 mcg/ml. The item description from the hospital states "OCTREOTIDE 50MCG/ML SOLUTION".

Please advise how to display the unit of measure and drug type measurement for the example above. MCG is not a valid value provided. Would the unit of measure indicate "1" and the drug type measurement as "ML"?

**Comment by SMKaweahHealth** [comment]
Agree, it is unclear what data is actually required. My EMR export would include Strength, Strength Unit, Volume, & Volume Unit. Those values won't apply to all products and it isn't clear how to determine which of those values are wanted for a given product.

**Reply by nmayle** [reply]
I believe that the intent of these two fields was to capture the actual number of units and the name of the units that corresponds to the charge(s).  While these are well defined for a particular J-code or a particular NDC, in the current hospital files, one can clearly see that while the code is listed as a J-code, the actual charge value for some payers may be according to a different quantity and unit of measure.  It can be particularly unclear as to the unit corresponding to the charges if both an NDC and J-code is shared for the same item.

Here is an example from a hospital file:

Note that this unit variation is not by drug, but the units being used as the basis of the price vary from payer-plan to payer-plan.
Also, note that the charge based on different units is included in the Min and Max and that is like comparing the price of a bushel of apples with the price of one apple.


```
  {
            "Code": "J9271",
            "Description": "Injection, pembrolizumab, 1 mg",
            "Code Type": "CPT/HCPCS",
            "Type": "Outpatient",
            "RevCode": "",
            "Gross Charge": "17,382.26",
            "Discounted Cash Price": "8,691.13",
            "Min": "47.88",
            "Max": "15,644.04",
            "Aetna MDCR HMO": "50.65",
            "BCBS MDCR": "50.65",
            "United MDCR Adv": "50.65",
            "Wellcare of NC": "50.65",
            "Beech Street PPO": "15,644.04",
            "Evolutions Healthcare": "16,513.15",
            "Multiplan": "15,644.04"
        },
```

These new fields would allow the hospitals to specify the quantity and units for each separately, so the prices would be comparable.  It sounds like the list of valid values for units of measure is not complete.

**Comment by ACLubin** [comment]
I am looking forward to an update on this question. I am also unsure what values should be used.

**Comment by TCL2023** [comment]
Yes, clarification for this would be great.

**Comment by RidenourK** [comment, marked as answer]
@zprice22 @SMKaweahHealth @ACLubin @TCL2023 Thank you for your questions and your patience. 

Please refer to rows 15-21of the CSV Wide examples, rows 19-30 of the CSV Tall examples, and lines 197-444 of the JSON examples available on this repository for [examples](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples) of how you could encode drug unit and drug type of measurement.

**Comment by zprice22** [comment]
Thank you for the examples. Based on the examples, the unit of measurement appears to be consistent with the NDC measure qualifier. It is not aligned to the HCPCS billable units.
